spec/unit/rom/support/registry_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+require 'rom/support/registry'
+
+describe ROM::Registry do
+  subject(:registry) { registry_class.new(mars: mars) }
+
+  let(:mars) { double('mars') }
+
+  let(:registry_class) do
+    Class.new(ROM::Registry) do
+      def self.name
+        'Candy'
+      end
+    end
+  end
+
+  describe '#fetch' do
+    it 'has an alias to []' do
+      expect(registry.method(:fetch)).to eq(registry.method(:[]))
+    end
```

Comment:
It would be better to just have a spec example for `#[]` method and check its behavior.